templates/js-cypress/main.js

```diff
@@ -0,0 +1,62 @@
+import { Actor } from 'apify';
+import fs from 'fs';
+import cypress from 'cypress';
+import { globby } from 'globby';
+import log from '@apify/log';
+import 'console.table';
+
+await Actor.init();
+
+const input = await Actor.getInput();
+const runOneSpec = (spec) => cypress.run({ config: input, spec });
+
+log.info(`Running tests with following input: ${JSON.stringify(input)}`);
+
```

Comment:
The spacing and logical continuity of this feels weird to me. I'd go for something like:

```
const runOneSpec = (spec) => cypress.run({ config: input, spec });

await Actor.init();

const input = await Actor.getInput();
log.info(`Running tests with following input: ${JSON.stringify(input)}`);

const tests = await globby('./cypress/e2e/*-spec.cy.js');
log.info(`Getting tests: ${tests}`);
...
```

What do you think?